tgui/packages/tgui/interfaces/JobSelection.tsx

```diff
@@ -99,11 +99,10 @@ export const JobEntry: SFC<{
 
 export const JobSelection = (props, context) => {
   const { act, data } = useBackend<Data>(context);
-  let departments = JSON.parse(JSON.stringify(data.departments)); // Why the fuck is it so hard to clone objects properly in JS?!
   if (!data?.static_data?.departments) {
     return null; // Stop TGUI whitescreens!
   }
```

Comment:
Why is this even necessary? If you're sending a `static_data` field , it should already be available from the first render. Also for simplicity's sake, don't nest objects like that, just use a differently named field like `departments_static`.

Reply:
Dunno, but it randomly sends null static data when using `tgui_dev`

Reply:
Yea, this code didn't even work, it errors out elsewhere, It was temporary code anyways.